### PR TITLE
Make buoys float again

### PR DESCRIPTION
**Pull request by mole99**
Until now buoys would just fall down, we need to apply the force with an impulse to make it work. To update the objects in debug mode, a tickPhysics function had to be created, like the vehicles already use.

### CODE REVIEW

**Comment by ShFil119** [comment]
After some testing, it works fine for buoys.

As street lights are also InstanceObjects doesn't it cause side effects?
screenshot_20180506_004749
Actually I don't have physical access to windows. :(
 Does street lights also float on water?

**Comment by mole99** [comment]
You are right, I haven't even thought about that!

With this change only buoys will float on water, later on we can add other objects which should float, but the issue for now was only concerning the buoys.

Unfortunately I had to hardcode the model name, but this already happens with the IslandLOD and should be therefore okay.

I do my best to improve this project ;)

File: rwengine/src/objects/InstanceObject.cpp [hunk 1]
```diff
@@ -64,57 +75,62 @@ void InstanceObject::tick(float dt) {
...
-        if (hI < NO_WATER_INDEX) {
-            wH = engine->data->waterHeights[hI];
-            wH += engine->data->getWaveHeightAt(ws);
-            inWater = vH <= wH;
-        } else {
-            inWater = false;
+
+    // Only certain objects should float on water
+    if ( floating )
+    {
```
Comment (ShFil119) [inline rwengine/src/objects/InstanceObject.cpp]: Can you reformat this?

**Comment by darkf** [comment]
Thanks

**Pull request merged**